app/Http/Controllers/TaskCategoriesController.php

```diff
@@ -70,6 +71,8 @@ public function show($id)
 
     public function edit(TaskCategory $taskCategory)
     {
+        Gate::authorize('view', $taskCategory);
```

Comment:
Instead of `Gate::authorize()`, you can use `$this->authorize()`. Please make this change and everything else is approved.

Reply:
ok sir.